testdata/scripts/config_noconfig.txt

```diff
@@ -1,28 +1,22 @@
 env HOME=$WORK/home
 
 # Gunk generate with out specified in config
-gunk generate ./gunk
-
-# Check that that files were generated in the correct folder
-exists gunk/all.pb.go gunk/all_pb2.py
+! gunk generate ./gunk
+stderr 'unable to load gunkconfig: no .gunkconfig found'
 
 -- go.mod --
 module testdata.tld/util
 
--- .gunkconfig --
-[generate]
-command=protoc-gen-go
-plugins=grpc
-
-[generate]
-protoc=python
-
 -- gunk/doc.go --
 package util // make this directory a Go package
 
 -- gunk/util.gunk --
 package util
 
+import (
```

Comment:
This is showing the wrong diff I believe. I changed `testdata/scripts/config_parent.txt` -> `testdata/scripts/config_projectroot.txt` so I am unsure why it is showing a diff between these two files.